FurCoNZ/Views/Order/Index.cshtml

```diff
@@ -19,10 +22,10 @@
                 <td>@ticketType.Value.TotalAvailable remaining</td>
                 <td>
                     <input asp-for="AvailableTicketTypes[ticketType.Key].QuantityOrdered" min="0" max="@ticketType.Value.TotalAvailable" />
-                    @*<input type="number" name="ticketnumber[@ticketType.Key]" value="0" min="0" max="@ticketType.Value.TotalAvailable" />*@
                 </td>
             </tr>
         }
+        </tbody>
     </table>
     <input type="submit" name="submit" value="Details >" />

```

Comment:
Instead of `<input />` for submit buttons, stick to bootstrap's button element (e.g. `<button clss="btn" type="submit">` that is themed up already
https://getbootstrap.com/docs/4.3/components/buttons/#examples

Reply:
:+1: